### PR TITLE
Add traceability section to issue templates

### PR DESCRIPTION
**Pull request by JaWetter**
## Risk analysis

- [x] Risks of the implementation in this pull request have been analyzed with following results:
- No significant risks. 

## Significance

- [x] Analysis, is the software change considered significant (ref. [MDCG 2020-3, chart C](https://drive.google.com/file/d/1QAqBhV-6NzuqXCY20kH5cH0VTGl83Jj0/view?usp=sharing)), has been made with following result: 
- **Not significant change**, can be made to the product certified according MDD.
	- Reasons for the decision - Just and update to the GitHub issue template, no effects on the product.

## Security check-up

_Instruction: Check that the OWASP top 10 have been taken into account._

[https://github.com/OWASP/www-project-top-ten/blob/master/index.md](https://github.com/OWASP/www-project-top-ten/blob/master/index.md)
- [x] **A01:2021-Broken Access Control** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A02:2021-Cryptographic Failures** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A03:2021-Injection** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A04:2021-Insecure Design** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A05:2021-Security Misconfiguration** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A06:2021-Vulnerable and Outdated Components** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A07:2021-Identification and Authentication Failures** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A08:2021-Software and Data Integrity Failures** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A09:2021-Security Logging and Monitoring Failures** - Just an update to the GitHub issue template, no effects on the software.
- [x] **A10:2021-Server-Side Request Forgery** - Just an update to the GitHub issue template, no effects on the software.

### CODE REVIEW

**Comment by JaWetter** [comment]
part of issue [#377](https://github.com/Sensotrend/sensotrend-regulatory/issues/377)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -47,3 +47,8 @@ To be verified by following cases:
 - [ ] 1. Pass/Fail (your name, SW version, date)
 - [ ] 2. Pass/Fail (your name, SW version, date)
 
+## Traceability
+
+_instruction: Check that the pull request is linked to an issue, if not then link it_
```
Comment (mrinnetmaki) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Shouldn't the instruction be the other way around, asking to check that the issue is linked to at least one pull request?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -37,3 +37,8 @@ To be verified by following test cases:
 - [ ] 1. Pass/Fail (SW version, date, verifier)
 - [ ] 2. Pass/Fail (SW version, date, verifier)
 
+## Traceability
+
+_instruction: Check that the pull request is linked to an issue, if not then link it_
```
Comment (mrinnetmaki) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Shouldn't the instruction be the other way around, asking to check that the issue is linked to at least one pull request?

**Review by mrinnetmaki — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (JaWetter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: yes, seems to be the older instructions were left unchanged, I have now fixed the wording.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -49,6 +49,6 @@ To be verified by following cases:
 
 ## Traceability
 
-_instruction: Check that the pull request is linked to an issue, if not then link it_
+_instruction: Check that the issue is linked to atleast one pull request, if not then link it_
```
Comment (mrinnetmaki) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `atleast` -> `at least`.

**Review by mrinnetmaki — CHANGES_REQUESTED** [review]

**Review by mrinnetmaki — APPROVED** [review]

**Pull request merged**